faseb-journal.csl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with=", " names-delimiter=", " page-range-format="expanded" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>FASEB Journal</title>
+    <title-short>FASEB</title-short>
```

Comment:
let's not put that in title-short, no (because it's not short ;). It'd be nice in ```summary``` though, even though that's not searched on the repository